packages/debug/src/browser/view/debug-view-model.ts

```diff
@@ -62,62 +57,36 @@ export class DebugViewModel implements Disposable {
         this.onDidChangeWatchExpressionsEmitter
     );
 
-    @inject(DebugViewOptions)
-    protected readonly options: DebugViewOptions;
-
     @inject(DebugSessionManager)
     protected readonly manager: DebugSessionManager;
 
     @inject(DebugWatchManager)
     protected readonly watch: DebugWatchManager;
 
-    protected readonly _sessions = new Set<DebugSession>();
+    // protected readonly _sessions = new Set<DebugSession>();
```

Comment:
Minor: I don't think it makes sense to preserver this line for historic reasons. Leftover?